docs/docs/genai/prompt-version-mgmt/version-tracking/track-application-versions-with-mlflow.mdx

```diff
@@ -3,249 +3,373 @@ title: Track Application Versions with MLflow
 description: Learn how to version GenAI applications using MLflow LoggedModel as a metadata hub, linking external code with traces and evaluations via mlflow.set_active_model().
 ---
 
+import FeatureHighlights from "@site/src/components/FeatureHighlights";
+import ConceptOverview from "@site/src/components/ConceptOverview";
+import TilesGrid from "@site/src/components/TilesGrid";
+import TileCard from "@site/src/components/TileCard";
 import { APILink } from "@site/src/components/APILink";
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
-
-# Track Application Versions with Externally Managed Code
-
-This guide demonstrates MLflow's primary approach for versioning GenAI applications when your core application code resides in an external version control system (VCS) like Git. In this workflow, an MLflow `LoggedModel` acts as a **metadata hub**, linking a conceptual application version to its specific external code (e.g., a Git commit), configurations, and associated MLflow entities like traces and evaluation results.
-
-The `mlflow.set_active_model()` function is key to this process, establishing a context so that subsequent traces and evaluations are automatically associated with the correct application version.
-
-## Prerequisites
-
-This guide requires the following packages:
-
-- **mlflow>=3.1**: Core MLflow functionality with GenAI features. MLflow 2.x is not supported as it lacks the version tracking features with `LoggedModel`.
-- **openai>=1.0.0**: Required for the OpenAI-based examples in this guide (if using other LLM providers, install their respective SDKs instead)
+import { GitBranch, Code2, Database, Settings, Eye, Target, Zap, BarChart3, PlayCircle } from "lucide-react";
+
+# Track Application Versions with External Code Management
```

Comment:
Do we need to use two types of cards? Can we use the first card type?

<img width="880" height="779" alt="Image" src="https://github.com/user-attachments/assets/470f4270-37a8-46d8-99ee-eab3bdb00cec" />